controller/src/main/resources/xsd/measurement.xsd

```diff
@@ -0,0 +1,64 @@
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           targetNamespace="http://uni.miskolc.ips.ilona.measurement/measurement"
+           elementFormDefault="qualified"
+           xmlns:pos="http://uni.miskolc.ips.ilona.measurement/position">
+
+    <xs:import namespace="http://uni.miskolc.ips.ilona.measurement/position" schemaLocation="position.xsd"/>
+
+
+    <xs:complexType name="MeasurementDTO">
+        <xs:sequence>
+            <xs:element name="id" type="xs:string"/>
+            <xs:element name="timestamp" type="xs:dateTime"/>
+            <xs:element name="position" type="pos:PositionDTO"/>
+            <xs:element name="wifiRSSI" minOccurs="0" maxOccurs="1">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="ap" minOccurs="1" maxOccurs="unbounded">
+                            <xs:complexType>
+                                <xs:simpleContent>
+                                    <xs:extension base="xs:int">
+                                        <xs:attribute name="ssid" type="xs:string" use="required"/>
+                                    </xs:extension>
+                                </xs:simpleContent>
+                            </xs:complexType>
+                        </xs:element>
+                    </xs:sequence>
+                </xs:complexType>
+            </xs:element>
+            <xs:element name="magnetometer" minOccurs="0" maxOccurs="1">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="xAxis" type="xs:double"/>
+                        <xs:element name="yAxis" type="xs:double"/>
+                        <xs:element name="zAxis" type="xs:double"/>
+                        <xs:element name="radian" type="xs:double"/>
```

Comment:
radian is not used in the real application, because the coordinates are adjusted by it.

Reply:
it is part of the the model so it is represented

Model may require refactoring :smiley: